### PR TITLE
[vcpkg] Add debug_message to vcpkg_configure_meson.cmake

### PR DESCRIPTION
**Pull request by kafeg**
Just simplify debugging when you are working on any port depended on meson and Mac M1.

### CODE REVIEW

**Review by PhoebeHui — APPROVED** [review]
LGTM, thanks for the PR!

cc @Neumann-A

File: scripts/cmake/vcpkg_configure_meson.cmake [hunk 1]
```diff
@@ -219,6 +219,9 @@ function(vcpkg_internal_meson_generate_cross_file _additional_binaries) #https:/
             COMMAND uname -m
             OUTPUT_VARIABLE MACHINE
             COMMAND_ERROR_IS_FATAL ANY)
+        
+        # show real machine architecture to visual understand, does we now in native Apple Silicon Terminal or inside the Rosetta
```
Comment (Hoikas) [inline scripts/cmake/vcpkg_configure_meson.cmake:223]: Can we please correct the grammar on this before merging?

File: scripts/cmake/vcpkg_configure_meson.cmake [hunk 2]
```diff
@@ -219,6 +219,9 @@ function(vcpkg_internal_meson_generate_cross_file _additional_binaries) #https:/
             COMMAND uname -m
             OUTPUT_VARIABLE MACHINE
             COMMAND_ERROR_IS_FATAL ANY)
+        
+        # show real machine architecture to visual understand, do we now in native Apple Silicon Terminal or inside the Rosetta
```
Comment (Deledrius) [inline scripts/cmake/vcpkg_configure_meson.cmake:223]:
```suggestion
        # Show real machine architecture to visually understand whether we are in a native Apple Silicon terminal or running under Rosetta emulation
```

**Review by Deledrius — COMMENTED** [review]
This might be a bit more readable.  The original is very difficult to parse in English.

File: scripts/cmake/vcpkg_configure_meson.cmake (hunk 2 above)
Comment (PhoebeHui) [reply scripts/cmake/vcpkg_configure_meson.cmake:223]: Thanks @Deledrius for your review! @kafeg, could you address the suggestion?

**Review by PhoebeHui — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**